components/automate-ui/src/app/pages/project/details/project-details.component.ts

```diff
@@ -69,28 +69,29 @@ export class ProjectDetailsComponent implements OnInit, OnDestroy {
 
     combineLatest([
       this.store.select(getStatus),
-      this.store.select(updateStatus)
+      this.store.select(updateStatus),
+      this.store.select(getAllStatus)
     ]).pipe(
-      takeUntil(this.isDestroyed),
-      map(([gStatus, uStatus]) => {
+      takeUntil(this.isDestroyed)
+    ).subscribe(([getProjectSt, updateSt, getRulesSt]) => {
         this.isLoading =
-          (gStatus !== EntityStatus.loadingSuccess) ||
-          (uStatus === EntityStatus.loading);
-      })
-    ).subscribe();
+          (getProjectSt !== EntityStatus.loadingSuccess) ||
+          (updateSt === EntityStatus.loading) ||
+          (getRulesSt !== EntityStatus.loadingSuccess);
+      });
 
     this.store.select(projectFromRoute).pipe(
       filter(identity),
-      takeUntil(this.isDestroyed),
-      map((state) => {
+      takeUntil(this.isDestroyed)
+      ).subscribe((state) => {
         this.project = <Project>Object.assign({}, state);
```

Comment:
While we're at it, let's either do
```suggestion
        this.project = state;
```
or 
```suggestion
        this.project = { ...state };
```
If we want a copy.